src/aggregation/bucket/range.rs

```diff
@@ -51,13 +49,16 @@ use crate::{DocId, TantivyError};
 ///     }
 /// }
 /// ```
-#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+#[derive(Clone, Debug, Default, PartialEq, Serialize, Deserialize)]
 pub struct RangeAggregation {
     /// The field to aggregate on.
     pub field: String,
     /// Note that this aggregation includes the from value and excludes the to value for each
     /// range. Extra buckets will be created until the first to, and last from, if necessary.
     pub ranges: Vec<RangeAggregationRange>,
+    /// Whether to return the buckets as a hash map
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub keyed: Option<bool>,
```

Comment:
why `Option<bool>,` instead of `bool`

Reply:
fixed in https://github.com/quickwit-oss/tantivy/pull/1424/commits/5ab5f070edb5ae7e2f00daeb626e517e8290d83a